src/routes/+layout.svelte

```diff
@@ -20,7 +20,7 @@
     import {browser, dev} from '$app/environment'
     import CookiesDisclaimerModal from '$lib/frontend/components/CookiesDisclaimerModal.svelte'
     import {initialLocale, loadTranslations, locale} from '$translations/index'
-    import { inject } from '@vercel/analytics'
+    import {inject} from '@vercel/analytics'
```

Comment:
There still seem to be some differences between your code style and ours. Can you import the Project.xml into your IDE as coding style?

Reply:
You are right, it looks good now